index.bs

```diff
@@ -168,17 +169,17 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
 
 <h3 id="image-item-properties">Image Item Properties</h3>
 
-<h4 id ="av1-configuration-item-property">AV1 Item Configuration Property</h4>
+<h4 id="av1-item-configuration-property">AV1 Item Configuration Property</h4>
 
 <pre class="def">
-  Box Type:                 <dfn value export for="AV1 Item Configuration Property">av1C</dfn>
+  Box Type:                 <dfn export for="AV1ItemConfigurationProperty">av1C</dfn>
   Property type:            Descriptive item property
   Container:                ItemPropertyContainerBox
   Mandatory (per  item):    Yes, for an image item of type 'av01'
   Quantity:                 One for an image item of type 'av01'
 </pre>
 
-The syntax and semantics of the <dfn>AV1 Item Configuration Property</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
+The syntax and semantics of the <dfn export>AV1ItemConfigurationProperty</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
```

Comment:
Should this be `[=AV1ItemConfigurationProperty=]` as it is already defined at line 175? What happens when there are multiple `<dfn>` for the same token? I looked at the [doc](https://speced.github.io/bikeshed/) but it was not clear.

Reply:
I had to really dig into the BikeShed documentation and mess around with this to understand how it works. My understanding is that the definition on line 175 defines the term `AV1ItemConfigurationProperty/av1C` (i.e. `av1C` in the context of `AV1ItemConfigurationProperty`) as opposed to `av1C` as used in the sample entry in AV1-ISOBMFF. That is done to uniquely distinguish between the two uses.

The <dfn> on line 182 defines the term `AV1ItemConfigurationProperty` itself.

I tried changing line 182 to `[=AV1ItemConfigurationProperty=]`, but that causes BikeShed to complain about missing definitions for `AV1ItemConfigurationProperty`.